### DISCUSSION TITLE
Better UI Framework Bootstrap or Tailwind ( Please give examples )

### DISCUSSION THREAD

**Discussion by IamNotaRobot99** [community/community#75919] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I'm relatively new to programming and currently practicing web development and as of the moment focusing in the front-end side, So I want to ask suggestion on which is better to use and learn.

**Comment by PriyanshNegi** [comment]
If you want to focus on quickly building projects with ready-made components and responsive design, Bootstrap might be a good starting point.

If you prefer a more customizable and utility-first approach, allowing you to have more control over the styles, consider learning Tailwind CSS.

If you are interested in or planning to learn React, exploring Material-UI can be beneficial, as it integrates well with React and provides a set of pre-built components.

**Comment by stolenfallen1** [comment, marked as answer]
It actually depends on what you want to achieve or build, do you want ready made component with a little to no customization and faster development then use bootstrap, but if you want to have more control then use tailwind. 

Learning both will not hurt actually since both there syntaxes are somewhat similar, but I personally think Bootstrap might be a better starting point than Tailwind for beginners. 

Whichever you choose, happy coding! 💯
